src/Schema/AST/ASTHelper.php

```diff
@@ -83,20 +78,21 @@ public static function mergeUniqueNodeList($original, $addition, bool $overwrite
     }
 
     /**
+     * Unwrap lists and non-nulls and get the name of the contained type.
+     *
      * @param  \GraphQL\Language\AST\Node  $definition
      * @return string
      */
     public static function getUnderlyingTypeName(Node $definition): string
     {
-        $type = $definition->type;
-        if ($type instanceof ListTypeNode || $type instanceof NonNullTypeNode) {
```

Comment:
Why is doing this not necessary anymore? Or was it never necessary?

Reply:
Somehow my comment got lost, so here it goes:

The `getUnderlyingNamedTypeNode` checks for those wrapping types already.